tests/phpunit/tests/option/sanitizeOption.php

```diff
@@ -69,6 +69,7 @@ public function sanitize_option_provider() {
 			array( 'timezone_string', 0, 0 ),
 			array( 'timezone_string', 'Europe/London', 'Europe/London' ),
 			array( 'timezone_string', get_option( 'timezone_string' ), 'invalid' ),
+			'Deprecated timezone string is accepted as valid (Trac#56468)' => array( 'timezone_string', 'America/Buenos_Aires', 'America/Buenos_Aires' ),
```

Comment:
😆 When you can't bring yourself to introduce a dataset without a key. I feel that.